src/Experimental/Executor/CoroutineExecutor.php

```diff
@@ -73,22 +74,22 @@ class CoroutineExecutor implements Runtime, ExecutorImplementation
     /** @var string|null */
     private $operationName;
 
-    /** @var Collector */
+    /** @var ?Collector */
```

Comment:
Not sure how `?` is supported in @var annotations, `|null` is afaik more standard way.

Reply:
I suppose it's a matter of opinion--there are other `?`-style instances in the code base that had nothing to do with me, and it is a bit more concise, but I have no preference, so I'll go with your idea.

Reply:
Just tested in Jetbrains IDE. It doesn't consider `?` a null on property